CONTRIBUTING.md

```diff
@@ -255,23 +256,24 @@ The following section provides instructions on what is needed to perform a Conju
 
 ### Run tests in supported platforms
 
-Before each release the following tests will need to be performed:
+Before each release the following steps need to be performed:
 
-- On *each* platform we support (macOS, RHEL 7/8, Windows), follow the instructions provided under [Running tests outside of a containerized environment](#running-tests-outside-of-a-containerized-environment).
+- Run tests against macOS and Windows environments. Follow the instructions provided under [Running tests outside of a containerized environment](#running-tests-outside-of-a-containerized-environment) for a detailed explanation on how to do this. The project's automation provides full test automation for RHEL 7/8 so tests do not need to be run again.
+  
+- Backwards compatibility - deploy Conjur Enterprise v5.6.3 and OSS v1.2.0 servers and
+  [run the integration test](#running-tests-outside-of-a-containerized) from *each* supported platform.
   
-- Run the integration tests against the following different Conjur server environments from any platform you choose:
+- Run the integration tests against the following different Conjur server environments from *any* platform you choose:
   
   1. An environment configured with a CA-signed certificate (can be configured at the Load Balancer level)
      
   1. An environment configured with a self-signed certificate
      
   1. An environment configured with an unknown CA certificate
 
-  Note environments used to pack the binary should not be the same environment to run the tests!
+  Note that environments used to pack the binary should not be the same environment to run the tests!
 
-- Backwards compatibility - deploy Conjur Enterprise v5.6.3 and OSS v1.2.0 servers and
-  [run the integration test](#running-tests-outside-of-a-containerized) from each supported platform.
-  
+- For *each* platform we support (Windows, macOS, RHEL 7/8), run basic sanity tests (`init`, `login`, `list`)
```

Comment:
Need to bold **run basic sanity tests manually**

Reply:
I think italicizing the manual (_manually_) would be enough. WDYT? @eladkug